controllers/providers/dns/infoblox.go

```diff
@@ -68,7 +68,8 @@ func (p *InfobloxProvider) CreateZoneDelegationForExternalDNS(gslb *k8gbv1beta1.
 		if err != nil {
 			return err
 		}
-		if len(findZone.Ref) > 0 {
+		log.Debug().Msgf("NoSplitBrain: %v", p.config.NoSplitBrain)
```

Comment:
Imo the message is important enough to be on info log-level
```suggestion
		if p.config.NoSplitBrain {
			log.Info().Msg("Split-brain prevention handling is enabled")
		}
```